### PR TITLE
user-guide: new chapter about firstboot

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: docs/user-guide/08-firstboot.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+To execute a single command, create a oneshot systemd service unit:
+
+<Tabs values={tabValues} >
+
+<TabItem value="on-premises" >
+```toml
+name = "firsboot-single"
+
+[[customizations.files]
```
Comment (ochosi) [inline docs/user-guide/08-firstboot.md]:
```suggestion
[[customizations.files]]
```

Comment (ochosi) [reply docs/user-guide/08-firstboot.md]: This looks like a typo

**Review by ochosi — COMMENTED** [review]
Is this a typo? If so it's there thrice

**Review by ochosi — CHANGES_REQUESTED** [review]
I think there are some errors in the markdown (which is why the test-build fails).

**Comment by lzap** [comment]
Fixed.

**Comment by lzap** [comment]
Rebased @ochosi

**Review by ochosi — APPROVED** [review]
LGTM! Thank you <3

**Comment by lzap** [comment]
I am surprised it rendered fine, I was pretty sure I messed up, I mean, I would have to do `npm install` which I wanted to avoid... :-)

https://osbuild.org/docs/user-guide/firstboot

**Pull request merged**